lib/adapters/AbstractAdapter.js

```diff
@@ -18,17 +19,19 @@ class AbstractAdapter extends AbstractReaderWriter {
 	 * @public
 	 * @param {Object} parameters Parameters
 	 * @param {string} parameters.virBasePath Virtual base path
+	 * @param {string[]} [parameters.excludes] List of GLOB patterns to exclude
```

Comment:
I think we agreed on using `glob` instead of all uppercase `GLOB`.

Reply:
Correct, I forgot 😅